examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -0,0 +1,407 @@
+"""
+.. _ref_dcb_test_composite_delamination
+
+======================================================================
+Static simulation of double cantilever beam test via cohesive elements
+======================================================================
+
+This example is inspired to a classic double cantilever beam test commonly used
+to study mode I interfacial delamination of composite plates.
+
+Description
+===========
+
+Objective
+~~~~~~~~~
+
+This example shows how to use PyMAPDL to simulate delamination in
+composite materials. PyDPF modules are also used for the post-processing of the results
+
+Problem Figure
+~~~~~~~~~~~~~~
+.. image:: ../../../images/dcb_test.png
+   :width: 400
+   :alt: DCB experimental test setup
+
+Procedure
+~~~~~~~~~
+* Launch MAPDL instance
+* Setup the model
+* Solve the model
+* Plot results of interest using PyMAPDL
+* Plot results using PyDPF
+
+Additional Packages Used
+~~~~~~~~~~~~~~~~~~~~~~~~
+* `Matplotlib <https://matplotlib.org>`_ is used for plotting purposes.
+
```

Comment:
I also got an error when running it for the first time in a clean venv, saying it requires Pandas.
Just like matplotlib, it is in requirements_docs.txt but not in the general requirements.
Someone having just installed pymapdl would not necessarily know to install the doc requirements to run the examples.

Reply:
Very good point @PProfizi !! Please @rmanno91 can you add pandas to `Additional packages used`?

Reply:
done